bin/sapcli

```diff
@@ -86,6 +89,16 @@ def parse_command_line(argv):
                             help="SAP Secure Login Client library (e.g. "
                                  "/Applications/Secure Login Client.app/Contents/MacOS/lib/libsapcrypto.dylib")
 
+    arg_parser.add_argument(
+        "--rest-over-rfc",
+        action='store_true',
+        dest="rest_over_rfc",
+        default=os.getenv("SAP_REST_OVER_RFC")
+        not in [None, 'n', 'no', 'false', 'off'],
+        help=
+        "Prefer doing rest call over SAP RFC client (to use SNC or if HTTP port is firewalled)"
+    )
```

Comment:
Can you keep the format of the core around, please?

Reply:
Done.